docs/user-guide/configuration/index.md

```diff
@@ -32,28 +28,39 @@ You can access information about properties by hovering over the property name.
     <a href="#annotations"><span class="key">annotations</span>:
     <span class="key">beta.screwdriver.cd/my-annotation</span>: <span class="value">my-data</span></a>
 <a href="#jobs"><span class="key">jobs</span>:</a>
-    <a href="#main-job"><span class="key">main</span>:</a>
+      <span class="key">main</span>:
+        <a href="#requires"><span class="key">requires</span>: <span class="value">[~pr, ~commit, ~sd@123:main]</span></a>
```

Comment:
Can you mention join case as well `[~pr, ~commit, ~sd@123:main, A, B]`

Reply:
I tried updating the example. Is this okay?